website/docs/r/group.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: "azuread"
+page_title: "Azure Active Directory: azuread_group"
+sidebar_current: "docs-azuread-resource-azuread-group"
+description: |-
+  Manages a Group within Azure Active Directory.
+
+---
+
+# azuread_group
+
+Manages a Group within Azure Active Directory.
+
+-> **NOTE:** If you're authenticating using a Service Principal then it must have permissions to `Read and write all groups` within the `Windows Azure Active Directory` API.
+
+-> **NOTE:** Additionally, due to a limitation within the API, you have to assign **one** of the following Azure Active Directory Roles to the Service Principal to be able to delete Groups:
+
+* User Account Administrator
+* Company Administrator
+
+You can assign one of the required Azure Active Directory Roles with the **AzureAD PowerShell Module**, which is available for Windows PowerShell or in the Azure Cloud Shell. Please refer to [this documentation](https://docs.microsoft.com/en-us/powershell/module/azuread/add-azureaddirectoryrolemember) for more details.
```

Comment:
I initially tried to do this in CloudShell but couldn't get this working (& ended up having to spin up a Windows VM to grant these permissions in the end); whilst it's likely I did something wrong, is this definitely possible in CloudShell?